client/utils/currency/index.js

```diff
@@ -119,19 +119,38 @@ export const formatCurrency = (
 		amount /= 100;
 	}
 
+	const isNegative = amount < 0;
+	const positiveAmount = isNegative ? -1 * amount : amount;
 	const currency = getCurrency( currencyCode, baseCurrencyCode );
 
 	if ( currency === null ) {
-		return composeFallbackCurrency( amount, currencyCode, isZeroDecimal );
+		return (
+			( isNegative ? '-' : '' ) +
```

Comment:
**Non-blocking:** What do you think about moving this sign logic to a variable? We can reuse it on lines 139 and 146 as well.

Reply:
Addressed in [5985c28](https://github.com/Automattic/woocommerce-payments/pull/7935/commits/5985c289f382db41cb1ef53aa18c67150f507def).